modin/experimental/xgboost/xgboost_ray.py

```diff
@@ -219,8 +225,12 @@ def predict(self, booster: xgb.Booster, **kwargs):
         return get_node_ip_address(), predictions
 
 
+ModinXGBoostActor = ray.remote(_ModinXGBoostActor)
```

Comment:
what's the reason for the change?

Reply:
If we have decorated class:
```python
@ray.remote
class ModinXGBoostActor:
..
```
sphinx can't build this.

Reply:
Fixed by https://github.com/modin-project/modin/pull/2965/commits/d3909aeccd61e695bd0d5f903c9b9e9b9c5dd8f3